Sources/Commands/SwiftTestTool.swift

```diff
@@ -42,6 +42,69 @@ struct NoMatchingTestsWarning: DiagnosticData {
     )
 }
 
+/// Diagnostic error when a command is run without its requeried command.
+struct RequiredArgumentDiagnostic: DiagnosticData {
+    static let id = DiagnosticID(
+        type: RequiredArgumentDiagnostic.self,
+        name: "org.swift.diags.required-argument",
+        defaultBehavior: .error,
+        description: {
+            $0 <<< { "\($0.dependentArgument)" } <<< "must be used with" <<< { "\($0.requiredArgument)" }
+    }
+    )
+    
+    let requiredArgument: String
+    
+    let dependentArgument: String
+}
+
+/// Defines the relational operators
+enum RelationalOperator {
+    
+    case equal
+    case greater
+    case less
+    case greaterOrEqual
+    case lessOrEqual
+}
+
+extension RelationalOperator: CustomStringConvertible {
+    
+    var description: String {
+        switch self {
+        case .equal :
+            return "equal to"
+        case .greater:
+            return "greater than"
+        case .less:
+            return "less than"
+        case .greaterOrEqual:
+            return "greater than or equal to"
+        case .lessOrEqual:
+            return "less than or equal to"
+        }
+    }
+}
+
+
+/// Diagnostic error for unsatisfied Int parameter
+struct UnsatisfiedIntParameterDiagnostic: DiagnosticData {
```

Comment:
We don't need such a sophisticated diagnostic. A simple one that just says the value > 0 should be fine.

Reply:
It is better to start with a simpler one and upgrade when there is a requirement in future.